ios/Exponent/Kernel/Views/EXErrorView.m

```diff
@@ -7,14 +7,37 @@
 #import "EXKernelAppRecord.h"
 #import "EXUtil.h"
 
+// for creating a filled button background in iOS <15
+@interface UIImage (Utils)
```

Comment:
To be honest I'm not a big fun of extending the `UIImage` just for this one method — it could be a helper somewhere, even directly in `EXErrorView`. Generally it's better not to extend other classes if you don't have to, especially with a private declaration where the interface is not exposed in the header like here.